lib/GuideCard/styles.scss

```diff
@@ -6,6 +6,7 @@
   display: flex;
   flex-direction: row;
   font-size: $typo-size-base;
+  color: $neutral-base;
```

Comment:
This is the only thing I had to add to the guide card. `GuideCard` is not refactored in this PR.